### ISSUE TITLE
Rails error when doing Fill Stitch

### ISSUE THREAD

**Issue by olneyg** [inkstitch/inkstitch#2694]
I tried to preview and got an error on satin column, but the object is not a satin column, it is a fill stitch.  I deleted the object. and duplicated the same shape from a different layer, but still get the same error.  Also, can't figure out why I'm getting fill in the middle of the B.  It should be filling only the letter area.  I'm a new user so don't really know what I'm doing.  
![2024-01-19 13_20_12- IN BE MY VALENTINES FILE 6](https://github.com/inkstitch/inkstitch/assets/44677078/8a0c41d8-bda0-4d0a-89b2-2b094f39ffbc)
![Be My Valentine drawing 6 WITH ERROR](https://github.com/inkstitch/inkstitch/assets/44677078/c98817e5-41b4-409e-81be-27522c8410ca)

**Comment by gusvisser** [comment]
From what i can see you probably used inkscape text and used path breakapart then the text will fill like you have if you need to break apart fill objects consider using inkstitch fill tool function.Also reconsider the text being used  as in the svg  because the fill areas are only a 0.5mm wide and converting a fill to a satin takes multiple steps
https://inkstitch.org/docs/stitch-library/ i reccommend to look at the stitch library also to get a better understanding

**Comment by bkgoodman** [comment]
I got the same error. Per your previous comment - my object was a Dynamic Offset, which converting to a path fixed.

**Comment by olneyg** [comment]
First, I did not used text but traced a text picture, which I later deleted.  Now I now about the hide function.  
I did this because I could not figure out how to load the font into nkstitch.  
I have used combine and break  apart multiple times.   When I used combine my results were backwards so I was getting a lot of jumps.  Combine seems to go from the top down , not the bottom up in the objects lists.  But inkstitch uses the object on bottom first when stitching out,  so I needed to redo the order to get it right and not have threads all over the place.


As far as I know I never did combine on any of the filled objects; I only used that on line objects.  But I may have accidentally.  This is my first attempt.
I'm not sure what a dynamic offset means so I will review it.  
Thank you for your help.  
Gail Olney


Sent from Yahoo Mail on Android 
 
  On Sat, Jan 20, 2024 at 3:09 PM, Brad ***@***.***> wrote:   


I got the same error. Per your previous comment - my object was a Dynamic Offset, which converting to a path fixed.
